### ISSUE TITLE
EKF little glitches after TakeOff

### ISSUE THREAD

**Issue by andrea25** [diydrones/ardupilot#1840]
This afternoon I and my collegues was flying our copter.
The pilot was my collegue Noemi. 
EKF enabled.
She do the TakeOff in Loiter.
Then, 5-20 second after TakeOff we see a little "glitch", the copter suddently move fast on the right.

After this everything was normal, everything was right. 
Nothing change. 
All was stable and the we can complete a mission in auto mode.

-----------------

Another time:
EKF enabled.
I do the TakeOff in AltHold.
Then, 5-20 second after TakeOff we see a little "glitch", the copter suddently move fast and go up.

After this everything was normal, everything was right. 
Nothing change. 

------------

Takeoff apart, during the loiter mode the copter was very stable.
And the auto mode is very clean, very good.
I do not have any problem with EKF after tuning.

---------

We can repeat the same "glitch" every 10-15 TakeOff, and only after TakeOff.
Actually we do not have any flashlog that we can share in public. But we can take anyscreenshot if you need.

-------
Noemi's LOG Sample

RED and GREEN --> ATT DesRoll and Roll
BLUE --> EKF1 ROLL

![rollekferror](https://cloud.githubusercontent.com/assets/10176752/6009620/a28b043e-ab2b-11e4-9a0a-2f23976b45ad.jpg)

Does anyone have our same problem?
Does anyone can look further about this?

**Comment by rmackay9** [comment]
This is a duplicate of one of these two issues I think:
https://github.com/diydrones/ardupilot/issues/1494
https://github.com/diydrones/ardupilot/issues/1682
